docs/content/python/guides/ui/index.md

```diff
@@ -135,8 +135,7 @@ allowed to take more actions on the repository than Sully.
 With this relatively straightforward policy, it's easy to trace where
 the users' allowed actions come from. But `Oso.get_allowed_actions()`
 can be especially powerful with more complicated policies. For example,
-if we used Oso's [SQLAlchemy Roles library
-features](guides/roles/sqlalchemy),
+if we used Oso's [roles](guides/rbac),
 we could have a policy that looks like this instead:
```

Comment:
this isn't accurate

Reply:
Yeah guide is on the list for tomorrow to update.